From 088624145b75743ef59a3ea8584b3c08fb0e8b95 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Tue, 15 Sep 2015 16:59:30 +0800
Subject: [PATCH 1095/1221] MLK-11502 dts: imx7d-sdb: change ramp up delay for
 VDD_SD1 regulator

Currently the ramp up delay for VDD_SD1 regulator is 1s, the delay
is added in the following patch:

   commit 558c8cde973b9b2abf9a69e3be5a887d38185da3
   MLK-11277 dts: imx7d-sdb: add ramp up delay for VDD_SD1 regulator

Here we find this delay will also impact the sd1 resume time cost, and
1s is too much for the sd1 resume, so we try to reduce the delay time
of this VDD_SD1 regulator, finally 200ms pass our test.

This patch change the ramp up delay for VDD_SD1 regulator from 1s to
200ms.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index d62e772..696f9ba 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -90,7 +90,7 @@
 			regulator-min-microvolt = <3300000>;
 			regulator-max-microvolt = <3300000>;
 			gpio = <&gpio5 2 GPIO_ACTIVE_HIGH>;
-			startup-delay-us = <1000000>;
+			startup-delay-us = <200000>;
 			enable-active-high;
 		};
 
-- 
1.7.5.4

